tests/Fixer/ControlStructure/YodaStyleFixerTest.php

```diff
@@ -636,6 +636,10 @@ function foo() {
                 '<?php $a = $b = null === $c;',
                 '<?php $a = $b = $c === null;',
             ],
+            [
```

Comment:
please move these to the `public function provideFix80Cases(): \Generator` data provider so the CI will pass